CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
     clients and then add the new scopes `ADMIN` and `ME`. See the
     [migration notes] (docs/Migration.md#from-22-to-30) for further details.
 
+- Allow colons (:) as field separators for URNs of extensions, since this is
+  what the SCIM specification defines. Using periods (.) is still possible,
+  but will log a warning message. 
+
```

Comment:
Alright, let's leave the changelog entry as it is. But nevertheless, I think my [previous comment](https://github.com/osiam/osiam/pull/108#discussion_r49280862) is still valid.